examples/gno.land/p/demo/grc/grc20/examples_test.gno

```diff
@@ -1,18 +1,125 @@
 package grc20
 
-// XXX: write Examples
-
-func ExampleInit()                            {}
-func ExampleExposeBankForMaketxRunOrImports() {}
-func ExampleCustomTellerImpl()                {}
-func ExampleAllowance()                       {}
-func ExampleRealmBanker()                     {}
-func ExamplePrevRealmBanker()                 {}
-func ExampleAccountBanker()                   {}
-func ExampleTransfer()                        {}
-func ExampleApprove()                         {}
-func ExampleTransferFrom()                    {}
-func ExampleMint()                            {}
-func ExampleBurn()                            {}
+import (
+	"std"
+
+	"gno.land/p/demo/ufmt"
+)
+
+// ExampleInit demonstrates how to initialize a new token.
+func ExampleInit() {
```

Comment:
I would love to have those `Example` tests. However, since they are not yet supported, I believe we should avoid using them because the tests are not actually tested.

We can keep your PR on hold until we have the feature. Alternatively, you could comment out everything and add a note explaining that it is just for inspiration, not a real test.

Reply:
The intention was to showcase how to use the token lib as I have had some troubles in the past with using banker as there was a lack of examples, but you are right I will address this in a comment